test/helpers/run-command.js

```diff
@@ -1,5 +1,3 @@
-'use strict';
```

Comment:
When I first updated the blueprint, there were a bunch of eslint errors around using "use strict" with `sourceType: module`. But for some reason I'm not getting those anymore, so I'll put them back.